core/templates/dev/head/services/UserServiceSpec.ts

```diff
@@ -17,18 +17,79 @@
  */
 
 describe('User Service', function() {
-  var UserService, $httpBackend, UrlInterpolationService;
-
+  var UserService, $httpBackend, UrlInterpolationService,
+    UserInfoObjectFactory;
   beforeEach(angular.mock.module('oppia'));
   beforeEach(angular.mock.inject(function($injector) {
     UserService = $injector.get('UserService');
     UrlInterpolationService = $injector.get(
       'UrlInterpolationService');
+    UserInfoObjectFactory = $injector.get(
+      'UserInfoObjectFactory');
     $httpBackend = $injector.get('$httpBackend');
   }));
 
+
+  it('should be return userInfo data', function() {
+    // creating a test user for checking profile picture of user.
+    var sampleUserInfoBackendObject = {
+      is_moderator: false,
+      is_admin: false,
+      is_super_admin: false,
+      is_topic_manager: false,
+      can_create_collections: true,
+      preferred_site_language_code: null,
+      username: 'tester',
+      user_is_logged_in: true
+    };
+
+    $httpBackend.expect('GET', '/userinfohandler').respond(200,
```

Comment:
When contents of parens span more than one line, break after '('.

/cc @oppia/dev-workflow-team

Reply:
Done!